stories/displaydata.stories.js

```diff
@@ -0,0 +1,102 @@
+import DisplayData from "../src/components/DisplayData.svelte";
+
+const testData = {
+  name: "Test data",
+  metrics: [
+    {
```

Comment:
The reason why I had 21 metrics in this test data is the pagination bar only appears when there's more than 20 metrics.